docs/code-samples/getting-started/daml/Scripts/Holding.daml

```diff
@@ -141,3 +145,24 @@ createParty name = do
   userId <- validateUserId name
   createUser (User userId (Some party)) [CanActAs party]
   pure party
+
+-- | Create a holding factory with reference.
+createHoldingFactory :
+  forall t.
+  ( HasAgreement t
+  , HasTemplateTypeRep t
+  , HasToAnyTemplate t
```

Comment:
For the quickstarter I would make this as simple as possible, not sure we need the full generic function